app/src/main/java/com/example/thecocktailsapp/Res/CocktailsRepository.kt

```diff
@@ -7,17 +7,30 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.flow
 import javax.inject.Inject
 
+private const val TAG = "CocktailsRepository"
+
 interface CocktailsRepository{
-                fun getCocktails():
-                        Flow<UIState<List<CocktailsResponse>>>
```

Comment:
CocktailsResponse or CocktailInfo?

Reply:
lets do CocktailInfo , sorry about the confusion